kolibri/plugins/facility/assets/test/state/facilityConfig.spec.js

```diff
@@ -139,15 +138,13 @@ describe('facility config page actions', () => {
       return store.dispatch('facilityConfig/saveFacilityConfig').then(() => {
         expect(DatasetStub.getModel).toHaveBeenCalledWith(1000, {});
         expect(saveStub).toHaveBeenCalledWith(expect.objectContaining(expectedRequest), false);
-        expect(store.state.facilityConfig.notification).toEqual('SAVE_SUCCESS');
       });
     });
 
     it('when save fails', () => {
       const saveStub = DatasetStub.__getModelSaveReturns('heck no', true);
       return store.dispatch('facilityConfig/saveFacilityConfig').then(() => {
```

Comment:
This test needs to continue off the `catch` because the action doesn't handle errors anymore.

```suggestion
      return store.dispatch('facilityConfig/saveFacilityConfig').catch(() => {
```